LGB.py

```diff
@@ -11,6 +13,9 @@
 from sklearn.preprocessing import OneHotEncoder,LabelEncoder
 np.random.seed(2019)
 import time
+import sys
+sys.path.append("src")
```

Comment:
Why add this line?

Reply:
Offline sync to replace this with `from scr import Evaluation`.